linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pamcracklib/cracklib_accounts_password_pam_dcredit/bash/shared.sh

```diff
@@ -0,0 +1,16 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platorm_ol,multi_platform_rhv
```

Comment:
These enable platforms are wrong. They need to only be enable for `Red Hat Enterprise Linux 6`